conans/client/generators/text.py

```diff
@@ -115,17 +122,21 @@ def _loads_cpp_info(text):
             deps_cpp_info = DepsCppInfo()
             for dep, configs_cpp_info in data.items():
                 if dep is None:
-                    cpp_info = deps_cpp_info
+                    continue
```

Comment:
I would say this is also changing behavior, and it won't be recovering the data for the aggregated variables.
Why changing it? Seems risky.

Reply:
Aggregated variables are recovered and composed of the values gathered from each dependency. So no need to read them or recover, as they will be composed when requested.

Reply:
Please put a comment there: "None correspond to the aggregated values from all the dependencies, we can skip it because we are assigning the dependencies individually and can get the aggregated value"